### PR TITLE
Fixes #25841: Create an instance id on each root server

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/25841

We need a migration to create the instance ID file, but we also want the instance ID to be loaded from the file on Rudder init : 
* create a service that reads the file and that just expose the instance id, if it does not exist a random value should be created 
* the check takes the service and does the file creation if needed

### CODE REVIEW

**Comment by clarktsiory** [comment]
Commit modified

**Comment by clarktsiory** [comment]
Commit modified

**Comment by clarktsiory** [comment]
Commit modified

**Comment by clarktsiory** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -3305,13 +3305,18 @@ object RudderConfigInit {
     lazy val mainCampaignService = new MainCampaignService(campaignEventRepo, campaignRepo, uuidGen, 1, 1)
     lazy val jsonReportsAnalyzer = JSONReportsAnalyser(reportsRepository, propertyRepository)
 
+    lazy val instanceUuidPath    = root / "opt" / "rudder" / "etc" / "instance-id"
+    lazy val instanceIdGenerator = new InstanceIdGeneratorImpl()
+    lazy val instanceIdService   = InstanceIdService.make(instanceUuidPath, instanceIdGenerator).runNow
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala:3310]: We will want to expose it in the `RudderConfigApi` (in `rci = init()` and then add one line with trait signature and the corresponding service.

**Review by fanf — CHANGES_REQUESTED** [review]
Looks good, it just need to be exposed in `RudderConfigApi`

**Comment by clarktsiory** [comment]
PR rebased

**Pull request merged**